tests/Behat/DrupalContext.php

```diff
@@ -306,4 +310,40 @@ protected function getLanguageIdByName(string $name): string {
     throw new \InvalidArgumentException("Language '{$name}' not found.");
   }
 
+  /**
+   * Configure language negotiation to use URL suffixes.
+   *
+   * @BeforeScenario @language-negotiation-suffix
+   */
+  public function setUpLanguageNegotiationSuffix(): void {
+    \Drupal::service('module_installer')->install(['oe_multilingual_url_suffix']);
+
+    $config = \Drupal::configFactory()->get(LanguageNegotiationSetter::CONFIG_NAME);
+
+    // Replace the default prefix URL negotiator with our suffix Url negotiator
+    // on the interface negotiation methods.
+    $enabled_interface_methods = $config->get('negotiation.' . LanguageInterface::TYPE_INTERFACE . '.enabled');
+    $enabled_interface_methods[LanguageNegotiationUrlSuffix::METHOD_ID] = $enabled_interface_methods[LanguageNegotiationUrl::METHOD_ID] ?? -19;
+    unset($enabled_interface_methods[LanguageNegotiationUrl::METHOD_ID]);
+    $this->configContext->setConfig(LanguageNegotiationSetter::CONFIG_NAME, 'negotiation.' . LanguageInterface::TYPE_INTERFACE . '.enabled', $enabled_interface_methods);
+    $this->configContext->setConfig(LanguageNegotiationSetter::CONFIG_NAME, 'negotiation.' . LanguageInterface::TYPE_INTERFACE . '.method_weights', $enabled_interface_methods);
+
+    // Replace the default prefix URL negotiator with our suffix Url negotiator
+    // on the content negotiation methods.
+    $enabled_content_methods = $config->get('negotiation.' . LanguageInterface::TYPE_CONTENT . '.enabled');
+    $enabled_content_methods[LanguageNegotiationUrlSuffix::METHOD_ID] = $enabled_content_methods[LanguageNegotiationUrl::METHOD_ID] ?? -19;
+    unset($enabled_content_methods[LanguageNegotiationUrl::METHOD_ID]);
+    $this->configContext->setConfig(LanguageNegotiationSetter::CONFIG_NAME, 'negotiation.' . LanguageInterface::TYPE_CONTENT . '.enabled', $enabled_content_methods);
+    $this->configContext->setConfig(LanguageNegotiationSetter::CONFIG_NAME, 'negotiation.' . LanguageInterface::TYPE_CONTENT . '.method_weights', $enabled_content_methods);
```

Comment:
Can't we use our own service `LanguageNegotiationSetter`? See `oe_multilingual_install` for an example.

Reply:
I did this because we doing enabling of module and configurations only for executing related behat scenario. By ConfigContext we could be sure that after behat scenario executing we will have properly reverted configuration.